README.md

```diff
@@ -1,27 +1,63 @@
 # LogCheck API: Getting Started
 
-## General
+## Prerequisites
+- A **terminal program** (e.g. Terminal, iTerm) running a **sh-compatible shell** (e.g. Bash, ZSH)
+- A **scripting language**. Our examples use Python and Ruby, but you may use any scripting language that
+    supports making an HTTP GET requests with basic HTTP authentication.
+
+## Getting Started
+
+### 1. Generate credentials
 - Email support to request API access
-- Log in to the website https://www.logcheckapp.com
-- In the menu bar, select `Settings` in the drop down under your name
-- Select the API Tokens tab
-- Press the `Generate a new token` tab and MAKE SURE TO SAVE YOUR `TOKEN_ID` and `SECRET_KEY`
-- Save `TOKEN_ID` and `SECRET_KEY` in a `.env` file.
+- Log in to the website [https://www.logcheckapp.com](https://www.logcheckapp.com/login)
+- In the drop down under your name, select `Settings`
+- Select the `API Tokens` tab on the left
+- Select the `Generate a new token` button
+- Leave the resulting page open until **step 2**
 
-## Ruby
-Run the `example.rb` script in the `ruby` directory.
+### 2. Clone this repo
+
+From your shell:
+
+```sh
+cd ~
```

Comment:
If we don't have a need to be in the home directory we should leave this `cd ~` step out. Assume the user knows where they want to put it.